pkg/deployments/index.ts

```diff
@@ -28,7 +28,7 @@ export async function getBalancerContractAt(task: string, contract: string, addr
  * @param task ID of the task to look the ABI of the required contract
  * @param contract Name of the contract to looking the ABI of
  */
-export async function getBalancerContractAbi(task: string, contract: string): Promise<unknown[]> {
+export function getBalancerContractAbi(task: string, contract: string): unknown[] {
```

Comment:
We should update the readme, which contains docs for these, and add a small changelog entry.